rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -1810,6 +1810,94 @@ public void rskTxWaitingForSignature_uses_updateCollection_rskTxHash_before_rski
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test
+    public void rskTxWaitingForSignature_preRSKIP326_emitNewPegoutConfirmedEvent() throws IOException {
```

Comment:
```suggestion
    public void rskTxWaitingForSignature_postRSKIP326_emitNewPegoutConfirmedEvent() throws IOException {
```
It's a post activation test, right?

Reply:
We should do this same test pre RSKIP 326, to ensure that the event is not emmited